Bridgecraft/Utils.swift

```diff
@@ -13,27 +13,43 @@ enum BridgecraftError: Error {
 }
 
 @discardableResult
-func shell(_ command: String, args: [String]) throws -> String {
+func shell(_ command: String, args: [String], verbose: Bool = false) throws -> String {
+    
     let ps = Process()
     ps.launchPath = command
     ps.arguments = args
     
-    let pipe = Pipe()
-    ps.standardOutput = pipe
+    let outputPipe = Pipe()
+    let errorPipe = Pipe()
+    ps.standardOutput = outputPipe
+    ps.standardError = errorPipe
     
     ps.launch()
     
-    var buffer = Data()
-    while ps.isRunning {
-        buffer.append(pipe.fileHandleForReading.readDataToEndOfFile())
+    ps.waitUntilExit()
```

Comment:
I'm afraid `waitUntilExit` cannot be used here since it depends on a runloop being installed on the current thread

Reply:
Done